index.js

```diff
@@ -387,6 +392,10 @@ exports.tarpit_delay_msa = function (connection, delay, k) {
   return delay;
 };
 
+exports.should_we_skip = function (connection) {
+  return connection.remote.is_private === true || connection.notes.disable_karma === true
+};
```

Comment:
There's nothing wrong with this, but I'd prefer to see it implemented like this:
```js
exports.should_we_skip = function (connection) {
    if (connection.remote.is_private) return true;
    if (connection.notes.disable_karma) return true;
    return false;
};
```
This is dead simple to reason about and it's both easier and cleaner to alter when someone else wants to add the third case to `should_we_skip()`.